examples/stable-diffusion/README.md

```diff
@@ -520,6 +520,21 @@ python image_to_image_generation.py \
     --bf16
 ```
 
+### Depth to Image Generation
+
+Here is how to generate a depth2img-guided image generation using HPU graphs with BF16:
+
+```bash
+python depth_to_image_generation.py \
+    --model_name_or_path "stabilityai/stable-diffusion-2-depth" \
+    --prompts "two tigers" \
+    --base_image "img.png" \
```

Comment:
This is a local path, is it possible to have a path pointing to a hub or github address?